test/nadia_test.exs

```diff
@@ -5,10 +5,7 @@ defmodule NadiaTest do
   alias Nadia.Model.User
 
   setup_all do
-    unless Application.get_env(:nadia, :token) do
-      Application.put_env(:nadia, :token, "TEST_TOKEN")
-    end
-    ExVCR.Config.filter_sensitive_data("bot[^/]+/", "bot<TOKEN>/")
```

Comment:
Well, this line will affect the cassette recorded by ExVCR.

I know it'll disturb the recording of multiple token tests.
But it's convenient for other tests, so please don't remove it.

Maybe it's better to have a setting for multiple token tests only.